frontend/src/components/commons/ProgressNode.jsx

```diff
@@ -32,3 +55,5 @@ class ProgressNode extends Component {
 }
 
 export default ProgressNode;
+
+export { styles };
```

Comment:
I don't think you need to export this.

Reply:
right... forgot to remove that after a fix.... resolved in 588fb6b